bundles/binding/org.openhab.binding.xbmc/src/main/java/org/openhab/binding/xbmc/rpc/calls/InputBack.java

```diff
@@ -0,0 +1,43 @@
+/**
+ * Copyright (c) 2010-2016 by the respective copyright holders.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.openhab.binding.xbmc.rpc.calls;
+
+import java.util.Collections;
+import java.util.Map;
+
+import org.openhab.binding.xbmc.rpc.RpcCall;
+
+import com.ning.http.client.AsyncHttpClient;
+
+/**
+ * Input.Back RPC
+ *
+ * @author Il Plebs
+ * @since 1.5.0
+ */
+public class InputBack extends RpcCall {
```

Comment:
Rather than have all these classes which are exactly the same apart from the command/name, perhaps it would be better to have a single `XxxRpcCall` class with an enumerator for the command?